common.mk

```diff
@@ -1299,13 +1299,11 @@ gdb: miniruby$(EXEEXT) run.gdb PHONY
 gdb-ruby: $(PROGRAM) run.gdb PHONY
 	$(Q) $(RUNRUBY_COMMAND) $(RUNRUBY_DEBUGGER) -- $(RUNOPT0) $(TESTRUN_SCRIPT) $(RUNOPT)
 
-LLDB_INIT = command script import -r $(srcdir)/misc/lldb_cruby.py
-
 lldb: miniruby$(EXEEXT) PHONY
-	lldb -o '$(LLDB_INIT)' miniruby$(EXEEXT) -- $(RUNOPT0) $(TESTRUN_SCRIPT) $(RUNOPT)
+	lldb --local-lldbinit miniruby$(EXEEXT) -- $(RUNOPT0) $(TESTRUN_SCRIPT) $(RUNOPT)
```

Comment:
`--local-lldbinit` doesn't work if I have a build directory not in the root directory.